config/processors/api_log_audit_aws.ghe.conf

```diff
@@ -21,10 +21,13 @@ filter {
     rename => { "[tmp][http_verb]" => "http.request.method" }
     rename => { "[tmp][ident]" => "service.name" }
     rename => { "[tmp][timestamp]" => "event.created" }
-    rename => { "[tmp][host]" => "host.name" }
+    rename => { "[tmp][host]" => "log.source.hostname" }
```

Comment:
Badal, could you take updated version and then make the same changes with nested notation?